src/nvector/pthreads/nvector_pthreads.c

```diff
@@ -98,18 +110,18 @@ static void* VScaleBy_PT(void* thread_data);
 static void* Vaxpy_PT(void* thread_data);
```

Comment:
I didn't since to keep the diff more focused. The other ones were `N_V` which made it hard to know if they were public or internal. With these, at least its easy to tell.